package.json

```diff
@@ -270,12 +270,12 @@
   },
   "scripts": {
     "vscode:prepublish": "npm run compile",
-    "package": "vsce package",
-    "compile": "tsc -b",
-    "watch": "tsc -b -w",
-    "lint": "eslint src/**/*.ts",
-    "test-compile": "tsc -p ./",
-    "test": "npm run test-compile && node ./out/test/runTest.js"
+    "compile": "tsc -p ./",
```

Comment:
If I understand it correctly `-p` (unlike `-b`) would _not_ recompile the dependencies, am I right?

```
 -p FILE OR DIRECTORY, --project FILE OR DIRECTORY  Compile the project given the path to its configuration file, or to a folder with a 'tsconfig.json'.
 -b, --build                                        Build one or more projects and their dependencies, if out of date
```

I'm fine with it either way as I often usually blow away `node_modules` relatively regularly anyway when changing dependencies, so I'm not quite convinced that even `-b` does good enough job in terms of recompilation anyway - or maybe I just have Node.js set up in some wrong way.

Reply:
I changed these because that's what the VS Code Yeoman templates specify as the default: https://github.com/microsoft/vscode-generator-code/blob/c3dc4980adb4aeb6cede8393f356ddd715fd397b/generators/app/templates/ext-command-ts/package.json#L26-L32.

FWIW calling `tsc` directly will go away when we move to `esbuild` or to `webpack`. Then we get a whole other set of challenges with debugging minified code 😭